Modules/Relationships/test/Relationships.Domain.Tests/TestHelpers/TestData.cs

```diff
@@ -43,10 +43,13 @@ public static Relationship CreateRevokedRelationship()
         return relationship;
     }
 
-    public static Relationship CreateTerminatedRelationship()
+    public static Relationship CreateTerminatedRelationship(IdentityAddress? from = null, IdentityAddress? to = null)
     {
-        var relationship = CreateActiveRelationship();
-        relationship.XXXFakeTerminate(IDENTITY_1, DEVICE_1);
+        to ??= IDENTITY_2;
+        var template = new RelationshipTemplate(to, DEVICE_2, 999, null, []);
+        var relationship = new Relationship(template, from ?? IDENTITY_1, DEVICE_1, null, []);
+        relationship.Accept(to, DEVICE_2, []);
```

Comment:
Can be replaced with `CreateActiveRelationship`

Reply:
Good catch.

Changes made.